Frames.podspec

```diff
@@ -15,9 +15,7 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Source/**/*.swift'
   s.exclude_files = "Classes/Exclude"
-  s.resource_bundles = {
-    'Frames' => ['Source/Resources/**/*']
-  }
```

Comment:
fyi, this is a good PR for future versions, but if fixing in 4.0.2, make sure you have this change on last release branch (release/4.0.2), so we do not add unintended changes as well.